src/main/java/org/openmrs/isanteplus/performancedata/options/model/EncounterOption.java

```diff
@@ -0,0 +1,22 @@
+package org.openmrs.isanteplus.performancedata.options.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+@AllArgsConstructor
+public class EncounterOption {
+    private static final long ENCOUNTERS = 1;
+
+    @Getter
+    @Setter
+    private long encounters;
```

Comment:
it is rather not a collection of encounters but their number - shouldn't we change it?

Reply:
ok